### ISSUE TITLE
OLC Issues

### ISSUE THREAD

**Issue by grekan** [prophecymud/go-prophecy-issues#170]
Recently went to change some things in KoP hall. 

When oediting items, they often show all item types instead of the type they are. 

As a result I broke this obj:

![image](https://user-images.githubusercontent.com/30678526/81882145-f4388d00-955f-11ea-931b-0f60f0fc743d.png)

It was a container that was closable and lockable, but there's no fields for that. 

Under extra I would expect to see closable closed, but try as I might to add them I cannot. 

Basically objs work as is, but if you actually change them, you may likely break them. 

Redits seem safer, haven't gotten to medits yet.

**Comment by grekan** [comment]
So I found part of the issue. The values fields are not showing in oedit. I thought I could remember there used to be 2 container types and was doing some googling and I found containers have values. 

![image](https://user-images.githubusercontent.com/30678526/81883122-6dd17a80-9562-11ea-8502-a75264ae8436.png)


I set all of value1 and it's working. What it won't let me do now is tied value 2 to the key. It says there's no such item.

**Comment by grekan** [comment]
Okay I mostly figured this out, v0 is weight, v3 is item capacity, v4 is multiplier, and v2 is the key vnum

But when I try to set it to 2953, as in the above comment's screen shot it says there's no such item, seems if I put in a number above 0 it says that. I can put in text though which makes no sense and putting the key name obviously don't work. 

Being able to see the values would be helpful. 

Also all the ? commands are gone.

**Comment by grekan** [comment]
Yeah I can't get v2 to work for the container, it's lockable, but I can't set a key.
